components/vf-form/vf-form__fieldset/README.md

```diff
@@ -0,0 +1,59 @@
+👋 Hi component authour, you've read the docs, right?
```

Comment:
These first 12 lines of boilerplate got left in. I should probably add a note to delete this.

Reply:
or it's a pre-commit to check if it's there and ask to delete it in the cli?

Reply:
because search has the action in the button text - I think we can do without having the label visible. 

I've snuck in a slight CSS change for the inline version so the button lines up nicely.